AFQ/viz_libs/afq_plotly.py

```diff
@@ -0,0 +1,318 @@
+        The streamline information
+
+    affine : ndarray, optional
+       An affine transformation to apply to the streamlines before
+       visualization. Default: no transform.
+
+    bundle_dict : dict, optional
+        Keys are names of bundles and values are dicts that should include
+        a key `'uid'` with values as integers for selection from the trk
+        metadata. Default: bundles are either not identified, or identified
+        only as unique integers in the metadata.
+
+    bundle : str or int, optional
+        The name of a bundle to select from among the keys in `bundle_dict`
+        or an integer for selection from the trk metadata.
+
+    colors : dict or list
+        If this is a dict, keys are bundle names and values are RGB tuples.
+        If this is a list, each item is an RGB tuple. Defaults to a list
+        with Tableau 20 RGB values
+
+    background : tuple, optional
+        RGB values for the background. Default: (1, 1, 1), which is white
+        background.
+
+    scene : fury Scene object, optional
+        If provided, the visualization will be added to this Scene. Default:
+        Initialize a new Scene.
+
+    show : bool
+        Whether to provide an interactive html file.
+        Default: False
+
+    show_inline : bool
+        Whether to embed the visualization inline in a notebook. Only works
+        in the notebook context. Default: False.
+
+    Returns
+    -------
+    Fury Scene object
+    """
+
+    tg, streamlines = vut.tract_loader(trk, affine)
+
+    if figure is None:
+        figure = go.Figure()
+
+    figure.update_layout(plot_bgcolor=_color_arr2str(background))
+
+    for (sls, color, name) in \
+            vut.tract_generator(tg, streamlines, bundle, bundle_dict, colors):
+        _draw_streamlines(figure, sls, color, name, n_points=resample)
+
+    return _inline_interact(figure, interact, inline)
+
+
+def stop_creating_gifs():
+    try:
+        plotly.io.orca.shutdown_server()
+    except ValueError:
+        _orca_err()
+
+
+def _orca_err():
+    print("pyAFQ is trying to generate gifs using plotly. "
+          + "This requires orca, which cannot be installed via pip. "
+          + "See: https://github.com/plotly/orca \n"
+          + "Or consider using fury to visualize with pyAFQ instead")
+
+
+def create_gif(figure,
+               file_name,
+               n_frames=60,
+               zoom=2.5,
+               z_offset=0.5,
+               creating_many=False,
+               size=(600, 600),
+               rotate_forward=True):
+    tdir = tempfile.gettempdir()
+
+    for i in range(n_frames):
+        theta = (i * 6.28) / n_frames
+        camera = dict(
+            eye=dict(x=np.cos(theta) * zoom,
+                     y=np.sin(theta) * zoom, z=z_offset)
+        )
+        figure.update_layout(scene_camera=camera)
+        try:
+            figure.write_image(tdir + f"/tgif{i}.png")
+        except ValueError:
+            _orca_err()
+            return
```

Comment:
I think that we do want the error raised, but with an explanation:

```suggestion
        except 
            ValueError(_orca_err())
```